packages/gatsby-theme-api-docs/src/components/resource/method/parameters.js

```diff
@@ -160,6 +162,17 @@ function ParameterRow(props) {
               <Info>{props.parameter.additionalDescription}</Info>
             </div>
           )}
+          {props.parameter.default && (
+            <div>
+              <Info>
+                Default
```

Comment:
I'd add `Default: ` here and remove the `{` `}`  from the [info-value.js](https://github.com/commercetools/commercetools-docs-kit/pull/2026/files#diff-0adadd2d413fac55014359abd5a48ef13846be40584169c38e774a04af23d12a) file